lib/app.js

```diff
@@ -212,10 +231,21 @@ module.exports = {
       }
     });
   },
-
+  getStatus: function( appId, cb ){
+    refs.deviceApps.child(appId)
+    .child('public/runtime/status')
+    .on('value', function(resp){
+      if ( resp.exists() ){
+        cb(resp.val());
+      } else {
+        cb();
+      }
+    });
+  },
+  
   setStatus: function( appId, status, cb ){
-    if ( _.isNull(status.match(/active|inactive|error/)) ) {
-      console.error('status must be active, inactive or error');
+    if ( _.isNull(status.match(/active|inactive|pending|error/)) ) {
+      console.error('status must be active, inactive, pending or error');
```

Comment:
pending is set by install worker, so this wouldn't be caught here, but... for completeness sake